phys2bids/physio_obj.py

```diff
@@ -7,6 +7,7 @@
 from itertools import groupby
 
 import numpy as np
+from copy import deepcopy
```

Comment:
Could you move this import between logging and itertools please? It's a core module